<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1440132401154" stop="1440132424011" version="1.4.4">
    <name>Default suite</name>
    <title>Default suite : Default test</title>
    <test-cases>
        <test-case start="1440132420723" stop="1440132422545" status="passed">
            <name>testPackagesCollectionImagesPresent</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="UMESH-PC"/>
                <label name="thread" value="8288@UMESH-PC.main"/>
            </labels>
        </test-case>
        <test-case start="1440132422592" stop="1440132423974" status="broken">
            <name>testSendUsYourFeedback</name>
            <failure>
                <message>NullPointerException: null</message>
                <stack-trace>java.lang.NullPointerException
	at com.via.base.util.PopUpHandler.accept(PopUpHandler.java:8)
	at com.via.holiday.pages.HomePageHoliday.sendUsYourFeedback(HomePageHoliday.java:102)
	at com.via.holiday.pages.TestHomePageHoliday.testSendUsYourFeedback(TestHomePageHoliday.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="UMESH-PC"/>
                <label name="thread" value="8288@UMESH-PC.main"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
